gridstatus/ercot.py

```diff
@@ -432,42 +509,257 @@ def get_interconnection_queue(self, verbose=False):
 
         return queue
 
+    def get_spp(
```

Comment:
I believe this now handles historical data, if that's the case, we can add the `@support_date_range(frequency="1D")` decorator.

Reply:
Yes, will do